README.md

```diff
@@ -17,6 +17,13 @@ Generation is done with a dockerfile (docker command is included in the makefile
 ```bash
 make
 ```
+### Descriptor deployment
```

Comment:
This could instead be "Adding a new service" or "Services in descriptor" or something like that - it's not really about deploying a descriptor 😄

Reply:
Done